CMSgov/onemac
# OY2 5909 State User can validate waiver numbers against SEATool data

Story: https://qmacbis.atlassian.net/browse/OY2-5909
Endpoint: https://d1evzs4pbg9krm.cloudfront.net

Changes:
- Fixed bug in SubmissionForm where the changeRequest wasn’t getting passed properly
- Updated the BigMAC Sink so that it creates all ID patterns for Plan Types 122 and 123 (1915b and 1915c waivers)
- Updated submit to also create all ID patterns
- "Format Document" on submit made some changes, too

Note: no changes to package-exists were made - I believe adding the IDs means the business logic pieces can check against whatever parts they “know” they should look for with the same exact match function.

Test:
1. Login to: https://d1evzs4pbg9krm.cloudfront.net
2. Click to “Submit New Waiver”
3. Bug check: try to select waiver authority, then type in an existing waiver number - you should not get a console error anymore (to see the error, can look at develop branch: https://d2dr7dgo9g0124.cloudfront.net/ )
4. Current form validation should not change (will be visible after story 5866 is complete)
5. Submit a Waiver and note the ID
6. Access appropriate DynamoDB table and note that there are items for each piece of the ID  so MD.00.R03.M34  should have entries for MD.00.R03.M34  and MD.00.R03 and MD.00
